src/test/java/com/yelp/nrtsearch/server/warming/WarmerTest.java

```diff
@@ -161,23 +186,61 @@ private byte[] getWarmingBytes(List<String> queryStrings) throws IOException {
 
   private List<SearchRequest> getTestSearchRequests() {
     List<SearchRequest> testRequests = new ArrayList<>();
-    for (int i = 0; i < 2; i++) {
+    for (int i = 0; i < 4; i++) {
       SearchRequest searchRequest =
           SearchRequest.newBuilder()
               .setIndexName(index)
               .setQuery(
                   Query.newBuilder()
-                      .setTermQuery(TermQuery.newBuilder().setField("field" + i).build())
-                      .build())
+                      .setFunctionScoreQuery(
+                          FunctionScoreQuery.newBuilder()
+                              .setQuery(
+                                  Query.newBuilder()
+                                      .setTermQuery(TermQuery.newBuilder().setField("field" + i)))
+                              .setScript(Script.newBuilder().setLang("js").setSource("3 * 5"))))
               .build();
       testRequests.add(searchRequest);
     }
     return testRequests;
   }
 
+  private List<SearchRequest> getTestBasicSearchRequests() {
+    List<SearchRequest> testRequests = new ArrayList<>();
+    int i = 0;
+    for (; i < 2; i++) {
+      SearchRequest searchRequest =
+          SearchRequest.newBuilder()
+              .setIndexName(index)
+              .setQuery(
+                  Query.newBuilder().setTermQuery(TermQuery.newBuilder().setField("field" + i)))
+              .build();
+      testRequests.add(searchRequest);
+    }
+
+    for (; i < 4; i++) {
+      SearchRequest searchRequest =
+          SearchRequest.newBuilder()
+              .setIndexName(index)
+              .setQuery(
+                  Query.newBuilder()
+                      .setFunctionScoreQuery(
+                          FunctionScoreQuery.newBuilder()
+                              .setQuery(
+                                  Query.newBuilder()
+                                      .setTermQuery(TermQuery.newBuilder().setField("field" + i)))
+                              .setScript(Script.newBuilder().setLang("js").setSource("3 * 5"))))
+              .build();
+      testRequests.add(searchRequest);
+    }
+
+    return testRequests;
+  }
+
   private List<String> getTestSearchRequestsAsJsonStrings() {
     return List.of(
-        "{\"indexName\":\"test_index\",\"query\":{\"termQuery\":{\"field\":\"field0\"}}}",
-        "{\"indexName\":\"test_index\",\"query\":{\"termQuery\":{\"field\":\"field1\"}}}");
+        "{\"indexName\":\"test_index\",\"query\":{\"functionScoreQuery\":{\"query\":{\"termQuery\":{\"field\":\"field0\"}},\"script\":{\"lang\":\"js\",\"source\":\"3 * 5\"}}}}",
```

Comment:
Before we were testing the same query twice to test it being simplified or not. With the last commit change, we have 2 extra identical queries and I am just wondering what is being tested with these extra identical queries. Just wondering if we need them or if we can keep only two and simplify the test code by removing the extra for loops in getTestBasicSearchRequests.